### ISSUE TITLE
invalid known_action_type in the table interaction_claims

### ISSUE THREAD

**Issue by need47** [griffithlab/dgi-db#216]
In postgresql:

```sql
select count(*) from interaction_claims where known_action_type ~ 'ruby';
 count 
-------
  1192
```
maybe parsing error related to ruby/rail?

**Comment by ahwagner** [comment]
Hey @need47. First off, _thank you_ for engaging with us on GitHub. Community feedback like yours helps us improve the resource for everybody.

I looked into it, and it appears that this is a bug relating to the ChEMBL tsv importer, probably with the _ad hoc_ [known](https://github.com/griffithlab/dgi-db/blob/master/lib/genome/importers/chembl/chembl_tsv_importer.rb#L22-L27) function.

Since we've recently revised how we handle ChEMBL for drug groups anyway, this is a good time to revise this importer to take advantage of those changes.
